applications/crossbar/src/crossbar_maintenance.erl

```diff
@@ -1182,3 +1184,23 @@ set_app_screenshots(AppId, PathToScreenshotsFolder) ->
               || SShot <- filelib:wildcard(kz_term:to_list(PathToScreenshotsFolder) ++ "/*.png")
              ],
     update_screenshots(AppId, MA, SShots).
+
+-spec db_init() -> 'ok'.
+db_init() ->
+    kz_datamgr:suppress_change_notice(),
+    _ = kz_datamgr:revise_doc_from_file(?KZ_CONFIG_DB, ?APP, <<"views/system_configs.json">>),
+    _ = kz_datamgr:revise_doc_from_file(?KZ_MEDIA_DB, ?APP, "account/media.json"),
```

Comment:
Meant to be <<"account/media.json">> ?

Reply:
same thing. `kz_datamgr:revise_doc_from_file` does a `list_to_binary`

Reply:
@lazedo for consistency I would like these to match the datatype :)